### DISCUSSION TITLE
The "deepspeed" parameter of DeepSpeed OneBitAdam/ZeroOneAdam

### DISCUSSION THREAD

**Discussion by BlinkDL** [Lightning-AI/pytorch-lightning#13795] [DDP / multi-GPU / multi-node]
I am using DeepSpeed strategy at this moment, and I'd like to try DeepSpeed OneBitAdam/ZeroOneAdam, but initialize it in the python code (instead of the json config).

However, there is a "deepspeed" parameter in OneBitAdam/ZeroOneAdam (and you can't pass None, because the optimizer will call deepspeed.mpu):
https://deepspeed.readthedocs.io/en/latest/optimizers.html#zerooneadam-gpu

May I ask where I can find this "deepspeed" object inside DeepSpeedStrategy, so I can pass it to the optimizer?

I am using the following code to initialize DeepSpeedStrategy: 
```
trainer = Trainer(strategy=DeepSpeedStrategy(config='deepspeed.json'), devices=NUM_GPUS, accelerator="gpu", precision=16)
```

**Comment by zhuangxy** [comment]
hi @BlinkDL did  you solve this issue at last? Currently I'm trying to run RWKV-4-7B retrain using ZeroOneAdam, hit exact the same issue.
